packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -334,17 +326,10 @@ export class StorageService extends EventEmitter {
   }
 
   public async updatePeersList() {
-    const allUsers = this.getAllUsers()
-    const registeredUsers = this.getAllRegisteredUsers()
-    const peers = [...new Set(await getUsersAddresses(allUsers.concat(registeredUsers)))]
-    const community: InitCommunityPayload = await this.localDbService.get(LocalDBKeys.COMMUNITY)
-    const sortedPeers = await this.localDbService.getSortedPeers(peers)
-    if (sortedPeers.length > 0) {
-      community.peers = sortedPeers
-      await this.localDbService.put(LocalDBKeys.COMMUNITY, community)
```

Comment:
Community data (peer list) in localDB should still be updated, either here or on UPDATE_PEERS_LIST in connections-manager or somewhere else